topic-operator/src/test/java/io/strimzi/operator/topic/ZkTopicStoreTest.java

```diff
@@ -6,38 +6,28 @@
 
 import io.strimzi.operator.topic.zk.Zk;
 import io.strimzi.test.EmbeddedZooKeeper;
-import io.vertx.core.Promise;
 import io.vertx.core.Vertx;
 import io.vertx.junit5.Checkpoint;
-import io.vertx.junit5.VertxExtension;
 import io.vertx.junit5.VertxTestContext;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
 
 import java.io.IOException;
-import java.util.Collections;
 
-import static org.hamcrest.CoreMatchers.instanceOf;
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.nullValue;
-import static org.hamcrest.MatcherAssert.assertThat;
-
-@Disabled
-@ExtendWith(VertxExtension.class)
-public class ZkTopicStoreTest {
-
-    private EmbeddedZooKeeper zkServer;
+public class ZkTopicStoreTest extends TopicStoreTestBase {
```

Comment:
Why are we changing this?

Reply:
To not duplicate the tests -- I extracted them to TopicStoreTestBase class.